### ISSUE TITLE
Salesforce integration: requirements gathering

### ISSUE THREAD

**Issue by macintoshpie** [SEED-platform/seed#2738]
Gather requirements for this request
Nick to reach out to potential user.

**Comment by macintoshpie** [comment]
@nllong Can you remind me the status of this?

**Comment by nllong** [comment]
still waiting on the contact from which to extract requirements. We will be emailed when the contact is ready....

**Comment by macintoshpie** [comment]
Closing as no longer relevant. We can create a new issue when this picks up again
